static/js/vue-cdr-access/tests/unit/singleUseLink.spec.js

```diff
@@ -5,7 +5,7 @@ import translations from '@/translations';
 import moxios from 'moxios';
 
 const uuid = '9f7f3746-0237-4261-96a2-4b4765d4ae03';
-const response_date = { link: `https://test.edu`, expires: '24hrs', id: uuid };
+const response_date = { link: `https://test.edu`, key: `12345`, expires: '24hrs', id: uuid };
```

Comment:
It seems like "link" no longer exists, so we should remove it here to avoid confusion